libraries/s3/lib/abortMultipartUpload.ml

```diff
@@ -4,11 +4,12 @@ type input = AbortMultipartUploadRequest.t
 type output = AbortMultipartUploadOutput.t
 type error = Errors_internal.t
 let service = "s3"
-let to_http service region req =
+let to_http service region (req : input) =
```

Comment:
We infer the wrong type here if there is no annotation. It really shouldn't be necessary.